index.js

```diff
@@ -50,8 +58,20 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
-
+function checkIfAnyMovieHasRating(movies,rate) {
```

Comment:
```suggestion
function checkIfAnyMovieHasRating(movies, rate = "G") {
```
This is the syntax for setting a default value for a parameter.